app/helpers/reactjs_helper.rb

```diff
@@ -0,0 +1,8 @@
+module ReactjsHelper
+
+  def mount_react_component(name, selector, data)
```

Comment:
Here's an example of how the helper is used:

```erb
<% title _("Statistics"), _('Statistics') %>

<div id="statistics"></div>

<%= mount_react_component('StatisticsChartsList', '#statistics', @metadata.to_json)%>
```
The selector is a 'querySelector' that is used to located the html element into which the component is mounted and must match something specific in the html page. The name identifies the react component. Randomization in the helper wouldn't work. 

Considering the reactMounter implementation below, it might be clearer is I changed the helper parameter name from 'name' to 'component' or something like that.

```js
export function mount(component, selector, data) {

    const components = {
        StatisticsChartsList: {
            type: StatisticsChartsList,
            markup: <StatisticsChartsList data={data}/>
        }
    };

    const reactNode = document.querySelector(selector);

    if (reactNode) {
        ReactDOM.render(components[component].markup, reactNode);
    }
}
```